### ISSUE TITLE
False positive: 'layout-table' rule failure on tables with role=presentation

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#812]
'layout-table' rule fails for the following cases
```
    <table id="table-1" role="presentation">
        <thead>
            <tr>
                <th>
                    Name2
                </th>
                <th>
                    Type2
                </th>
            </tr>
        </thead>
    </table>

    <table id="table-2" role="presentation">
        <caption>Monthly savings caption2</caption>
        <tr>
            <td>
                <a href="#">t2 s1</a>
            </td>
            <td>
                <a href="#">t2 s2</a>
            </td>
        </tr>
        <tr>
            <td>January</td>
            <td>$100</td>
        </tr>
    </table>

    <table id="table-3" role="presentation" summary="SECOND- Monthly savings">
        <tr>
            <td>
                <a href="#">t2 s1</a>
            </td>
            <td>
                <a href="#">t2 s2</a>
            </td>
        </tr>
        <tr>
            <td>J1</td>
            <td>k1</td>
        </tr>
    </table


```

c.c @waabid

**Comment by WilcoFiers** [comment]
This is a duplicate of https://github.com/dequelabs/axe-core/issues/561. It needs to be tested to make sure that this doesn't actually trip up any AT in the baseline.

**Comment by iamrafan** [comment]
I tested these cases with Voiceover/OS X, JAWS/IE, NVDA/FF and Narrator/Edge. None of the combinations announce header, caption or summary used here

**Comment by WilcoFiers** [comment]
Cool, in that case lets change axe to allow it :)

**Comment by dylanb** [comment]
Suggest changing this to a best practice

**Comment by WilcoFiers** [comment]
I don't agree with that. The layout-table rule has an unfortunate name, but it genuinely tests for WCAG violations, e.g. not using `<th>` / `<caption>` on layout tables. Using `role="presentation"` makes those elements inert, but without it, that would be a violation.

**Comment by dylanb** [comment]
I am suggesting making it a best practice violation if there is a summary for example with role="presentation"

**Comment by WilcoFiers** [comment]
Ahh, gotcha. Maybe. I'd say that is less important though.

**Comment by jkodu** [comment]
@dylanb @WilcoFiers 
I will be picking this up.

**Comment by jkodu** [comment]
Microsoft is going to work/ resolve this issue.

**Comment by marcysutton** [comment]
This should be fixed with #828.
